commands/push.go

```diff
@@ -95,3 +104,20 @@ func pushStack(services *stack.Services, queueDepth int) {
 	wg.Wait()
 
 }
+
+func validateImages(services *stack.Services) string {
+	errMsg := make([]string, 0)
+	for name, function := range services.Functions {
+		if function.SkipBuild {
+			continue
+		}
+		ip := "(\\b\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\b)(:\\d+)?"
```

Comment:
My gut feeling is that this is too complicated. Why are we using a `regex` when a string/contains would probably suffice?

```
if there's a `/` in the string.. then go for it?
```

Reply:
That sounds like a logical check... I was obviously over thinking this!